book/special_variables.md

```diff
@@ -118,6 +118,37 @@ A list of directories which will be searched when using the `source`, `use`, or
 - [Module Path](./modules/using_modules.md#module-path)
 - [Configuration - `$NU_LIB_DIRS`](./configuration.md#nu-lib-dirs-constant)
 
+### `$env.NU_LOG_LEVEL`
+
+The [standard library](/book/standard_library.md) offers logging in `std/log`. The `NU_LOG_LEVEL` environment variable is used to define the log level being used.
```

Comment:
Is this - with the mention and reference of standard library - sufficient to clarify the conditional nature of it? That it is not available in a fresh shell session?

Reply:
I don't think it's specific enough.

```suggestion
The [standard library](/book/standard_library.md) offers logging in `std/log`. The `NU_LOG_LEVEL` environment variable is used to define the log level being used for custom commands, modules, and scripts.
```